sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -21,35 +23,58 @@ public static class TestEnvironment
 
+        /// <summary>
+        /// The name of the directory where the running assembly is located.
+        /// </summary>
+        /// <value>The name of the current working directory.</value>
+        private static string CurrentWorkingDirectory => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
         /// <summary>
         /// The relative path to the JPG file which contains the receipt to be used for tests.
         /// </summary>
         /// <value>The relative path to the JPG file.</value>
-        public static string ReceiptPath => Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), AssetsFolderName, ReceiptFilename);
+        public static string JpgReceiptPath => Path.Combine(CurrentWorkingDirectory, AssetsFolderName, JpgReceiptFilename);
+
+        /// <summary>
+        /// The relative path to the PNG file which contains the receipt to be used for tests.
+        /// </summary>
+        /// <value>The relative path to the PNG file.</value>
+        public static string PngReceiptPath => Path.Combine(CurrentWorkingDirectory, AssetsFolderName, PngReceiptFilename);
 
         /// <summary>
         /// The URI string to the JPG file which contains the receipt to be used for tests.
         /// </summary>
         /// <value>The URI string to the JPG file.</value>
-        public static string ReceiptUri => string.Format(FileUriFormat, AssetsFolderName, ReceiptFilename);
+        public static string JpgReceiptUri => string.Format(FileUriFormat, AssetsFolderName, JpgReceiptFilename);
 
         /// <summary>
-        /// Retrieves the relative path to a PDF form available in the test assets.
+        /// Retrieves the relative path to a PDF or TIFF form available in the test assets.
         /// </summary>
         /// <param name="index">The index to specify the form to be retrieved.</param>
-        /// <returns>The relative path to the PDF form corresponding to the specified index.</returns>
-        public static string RetrieveInvoicePath(int index)
+        /// <param name="contentType">The type of the form to be retrieved. Currently only PDF and TIFF are available.</param>
+        /// <returns>The relative path to the PDF or TIFF form corresponding to the specified index.</returns>
+        public static string RetrieveInvoicePath(int index, ContentType contentType)
         {
-            var filename = string.Format(InvoiceFilenameFormat, index);
-            return Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), AssetsFolderName, filename);
+            var extension = contentType switch
+            {
+                ContentType.Pdf => "pdf",
+                ContentType.Tiff => "tiff",
+                _ => throw new ArgumentException("The requested content type is not available.", nameof(contentType))
```

Comment:
I'm wondering whether we want to throw this exception in tests when we know what files we have available?  It's basically a developer exception for tests, where we're the developers...

Reply:
Well, we are not perfect.
We can make it default to PDF (or something else), but I wanted to avoid silent issues. They tend to be harder to debug.